File: PreChargedObjects.java
public class PreChargedObjects {

        public static void agregarElementos() {
                Employee employee1 = new Employee("Jafet", "Santoyo");
                Employee employee2 = new Employee("Alejandro", "Montejano");
                Employee employee3 = new Employee("Andrea", "Durán");
                Employee employee4 = new Employee("Atziri", "Mancilla");

                BankAccount account1 = new BankAccount(12345, 'A');
                BankAccount account2 = new BankAccount(45678, 'B');
                BankAccount account3 = new BankAccount(65445, 'A');
                BankAccount account4 = new BankAccount(16985, 'C');
                BankAccount account5 = new BankAccount(23784, 'A');
                BankAccount account6 = new BankAccount(79874, 'B');
                BankAccount account7 = new BankAccount(34977, 'C');
                BankAccount account8 = new BankAccount(32467, 'B');

                employee1.addAccount(account1);
                employee1.addAccount(account2);
                employee2.addAccount(account3);
                employee2.addAccount(account4);
                employee3.addAccount(account5);
                employee3.addAccount(account6);
                employee4.addAccount(account7);
                employee4.addAccount(account8);
                Bank.addAccountsList(account1);
                Bank.addAccountsList(account2);
                Bank.addAccountsList(account3);
                Bank.addAccountsList(account4);
                Bank.addAccountsList(account5);
                Bank.addAccountsList(account6);
                Bank.addAccountsList(account7);
                Bank.addAccountsList(account8);
                Bank.addEmployee(employee1);
                Bank.addEmployee(employee2);
                Bank.addEmployee(employee3);
                Bank.addEmployee(employee4);
        }
}

File: PreChargedObjects.py
from Employee import Employee
from BankAccount import BankAccount
from Bank import Bank

class PreChargedObjects:
    @staticmethod
    def agregar_elementos():
        employee1 = Employee("Jafet", "Santoyo")
        employee2 = Employee("Alejandro", "Montejano")
        employee3 = Employee("Andrea", "Durán")
        employee4 = Employee("Atziri", "Mancilla")

        account1 = BankAccount(12345, 'A')
        account2 = BankAccount(45678, 'B')
        account3 = BankAccount(65445, 'A')
        account4 = BankAccount(16985, 'C')
        account5 = BankAccount(23784, 'A')
        account6 = BankAccount(79874, 'B')
        account7 = BankAccount(34977, 'C')
        account8 = BankAccount(32467, 'B')

        employee1.add_account(account1)
        employee1.add_account(account2)
        employee2.add_account(account3)
        employee2.add_account(account4)
        employee3.add_account(account5)
        employee3.add_account(account6)
        employee4.add_account(account7)
        employee4.add_account(account8)
        
        Bank.add_accounts_list(account1)
        Bank.add_accounts_list(account2)
        Bank.add_accounts_list(account3)
        Bank.add_accounts_list(account4)
        Bank.add_accounts_list(account5)
        Bank.add_accounts_list(account6)
        Bank.add_accounts_list(account7)
        Bank.add_accounts_list(account8)
        
        Bank.add_employee(employee1)
        Bank.add_employee(employee2)
        Bank.add_employee(employee3)
        Bank.add_employee(employee4)
